dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldPathHelper.java

```diff
@@ -62,22 +66,20 @@
 {
     private final SchemaService schemaService;
 
-    public void apply( List<FieldPath> fieldPaths, Class<?> rootKlass )
+    public List<FieldPath> apply( List<FieldPath> fieldPaths, Class<?> rootKlass )
     {
-        if ( rootKlass == null || fieldPaths.isEmpty() )
+        if ( fieldPaths.isEmpty() || rootKlass == null )
         {
-            return;
+            return Collections.emptyList();
```

Comment:
Not super critical but in general we should prefer `List.of()` over the old `Collections.emptyList()`.